### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by suitupstan**
- added stats.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,11 @@
 - 📫 How to reach me: emmanuel.eban@conicapps.com
 - 🤝 Connect with me on LinkedIn: <https://www.linkedin.com/in/eebann/>
 
-<!---
-available!!
---->
+## 📈 GitHub Stats
+
+  <div align="right">
```
Comment (Panquesito7) [inline README.md:11]:
```suggestion
  <div align="center">
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (suitupstan) [reply README.md:11]: nah it is intentional

Comment (Panquesito7) [reply README.md:11]: If you center it, it'll look like this or similar.

image

What do you think?

**Comment by eebbann** [comment]
thanks..appreciate

**Pull request merged**